app/views/companies/index.html.erb

```diff
@@ -1,6 +1,13 @@
 <nav class="breadcrumb">
   <%= link_to 'Home', root_path, class: "breadcrumb-item small text-decoration-none" %>
-  <span class="breadcrumb-item active small ">Stock</span>
+  <% if params[:query] %>
+    <%= link_to 'Stock', companies_path, class: "breadcrumb-item small text-decoration-none" %>
+    <span class="breadcrumb-item active small">Search result</span>
+  <% elsif params[:stock] %>
+    <span class="breadcrumb-item active small "><%= params[:stock] %> Stock</span>
```

Comment:
Something to consider, maybe it does not make any sense, etc:

Wouldn't I be able to sent this link to as many people as I can `/companies?stock=You%20can%20donate%20to%201234566123121232%20bank%20account%20and%20help%20our%20service!` and trick the into donating money, etc.?

Reply:
in the other words, maybe it would make sense to accept only specific values for `stock` params :)

Reply:
Yes, very good idea.